internal/mq/mqimpl/rocksmq/server/rocksmq_impl.go

```diff
@@ -458,6 +458,39 @@ func (rmq *rocksmq) RegisterConsumer(consumer *Consumer) error {
 	return nil
 }
 
+func (rmq *rocksmq) GetLatestMsg(topicName, groupName string) (ConsumerMessage, error) {
+	if rmq.isClosed() {
+		return ConsumerMessage{}, errors.New(RmqNotServingErrMsg)
+	}
+
+	rmq.storeMu.Lock()
+	defer rmq.storeMu.Unlock()
+	key := constructCurrentID(topicName, groupName)
+	_, ok := rmq.consumersID.Load(key)
+	if !ok {
+		log.Warn("RocksMQ: channel " + key + " not exists")
+		return ConsumerMessage{}, fmt.Errorf("ConsumerGroup %s, channel %s not exists", groupName, topicName)
+	}
+
+	readOpts := gorocksdb.NewDefaultReadOptions()
+	defer readOpts.Destroy()
+	iter := rocksdbkv.NewRocksIterator(rmq.store, readOpts)
+	defer iter.Close()
+	iter.SeekToLast()
```

Comment:
The implementation here only works for single topic.
We need to add test cases for the following cases:
1) if a stream didn't write without any message, what is  the latest message return?
2)test with multiple topics.
Also the API should be GetLatestMsg(topicName) and it returned the latest written message for single topic.